stix_shifter_modules/nozomi_vantage/stix_transmission/connector.py

```diff
@@ -28,17 +28,19 @@ async def create_results_connection(self, query, offset, length, metadata=None):
         """
         data = []
         try:
-            result_count, total_records, page_number, page_size = self.handle_metadata_and_page_size(offset, length,
-                                                                                                     metadata)
+            offset = int(offset)
+            length = int(length)
+            result_count = offset
+            total_records, page_number, offset = self.handle_metadata(offset, length, metadata)
```

Comment:
Can you clean up some of these variable names? We have length, result_count, expected_result_count, total_records and local_result_count. It's difficult to follow what these are actually used for. 

You could add it as a comment if you'd like to keep the names.

Reply:
Inline comment are added.